flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/RoundRobinOperatorStateRepartitioner.java

```diff
@@ -488,13 +489,16 @@ private static final class GroupByStateNameResults {
         }
     }
 
-    private static final class StateEntry {
+    @VisibleForTesting
+    public static final class StateEntry {
```

Comment:
```suggestion
    static final class StateEntry {
```

A package-private access would be enough

Reply:
Resolved, thanks for your commtent~